.stylelintrc

```diff
@@ -0,0 +1,70 @@
+{
+  "extends": [
+    "stylelint-config-standard-scss",
+    "stylelint-config-idiomatic-order"
+  ],
+  "plugins": [
+    "stylelint-declaration-strict-value",
+    "@stylistic/stylelint-plugin"
+  ],
+  "overrides": [
+    {
+      "files": ["src/assets/scss/*.{css,scss}"],
+      "customSyntax": "postcss-scss",
+      "rules": {
+        "media-feature-range-notation": "prefix",
+        "color-function-notation": "legacy",
+        "shorthand-property-no-redundant-values": null,
+        "declaration-block-no-redundant-longhand-properties": [true, {
+          "ignoreShorthands": [
+            "inset",
+            "inset-block",
+            "inset-inline",
+            "mask",
+            "overscroll-behavior",
+            "padding-block",
+            "padding-inline",
+            "place-content",
+            "place-items",
+            "place-self",
+            "scroll-margin",
+            "border-block",
+            "border-block-end",
+            "border-block-start",
+            "border-inline",
+            "border-inline-end",
+            "border-inline-start"
```

Comment:
None of these are necessary

Reply:
They aren't necessary now, but I would like to keep them in. I want to set up the style lint so we don't have to remember to add these back if we use one of these properties.

Reply:
Gotcha. Were did this list come from?

Reply:
I've got the list from Fully Stacked at https://fullystacked.net/prefix/

Reply:
I got this list by going through the list of properties that this rule will fix ([which can be seen here](https://stylelint.io/user-guide/rules/declaration-block-no-redundant-longhand-properties/#ignoreshorthands-regex-regex-string)), and if the property isn't supported by all the major browsers, I prevent it from using it.

Reply:
Got it. I assume all the block and inline related styles are because we haven't yet been given permission to drop support for IE11? This is another place that could warrant a code comment if you switch the config file to a JS file.

Reply:
No, it was mostly due to Chrome just starting to support it in 2020.

I just did the calculation, and it is only 1.57% of global users use browsers that don't support these properties. so we could allow some of these stylles.